src/webhooks/webhook.js

```diff
@@ -25,17 +25,24 @@ aha.on("webhook", async ({ headers, payload }) => {
 async function handlePullRequest(payload) {
   const pr = payload.pull_request;
 
+  console.log("Linking pull request")
+
   // Make sure the PR is linked to its record.
   const record = await linkPullRequest(pr);
+  console.log("Done linking pull request")
+
 
   // Generate events.
   if (record) {
+    console.log("Generating events");
+    await triggerEvent("pr", payload, record);
+
     if (pr.head?.name) {
+      console.log("Linking branch")
       await linkBranch(pr.head.name, pr.repo.html_url);
     }
-
-    await triggerEvent("pr", payload, record);
   } else {
+    console.log("null record, triggering event anyway")
```

Comment:
I think we should word these triggering ones as "Trigger event with record" and "Triggering event without record"